### DISCUSSION TITLE
Multiple Module css/scss slectors

### DISCUSSION THREAD

**Discussion by BryceAMcDaniel** [vercel/next.js#12449] [Help] [answered]
Hello!
I was wondering if you could help with two use cases.
I'm using tailwind/postcss and would like to combine css-modules for styling custom sections using the .module.scss convention. Currently is there a way to select both custom-classNames and css-modules. I was expecting something like this to work, however, I wasn't able to get it to work:

```<div className={styles.base + "font-lg"}> ...```

Are we able to use multiple css-modules styles? to apply multiple selectors to one element?

```<div className={[styles.base, styles.btn]}> ...```

Thanks! and I'm quite liking the css-modules.

**Comment by rafaelalmeidatk** [comment, marked as answer]
`styles.base + "font-lg"` doesn't work because it would produce a string like `styles_basefont-lg` instead of `styles_base font-lg`, if you add a space it will work: `styles.base + " font-lg"`.

To make things cleaner you can create a function to transform multiple arguments to a string: 

```js
const cx = (...classNames) => classNames.join(' ');
```

Then you can use it like this: 
```js
<div className={cx(styles.base, styles.btn)}>
```

For more advanced rules (like conditional styles) it may be better to use something like the classnames library: https://github.com/JedWatson/classnames

**Reply by BryceAMcDaniel** [reply]
I swear I inspected the element and did not see the appended string on the class at the end!  

Thanks for the response, and this actually helps.

**Reply by peter-palmer** [reply]
next.js........ so complicated for such simple thing

**Reply by rafaelalmeidatk** [reply]
@peter-palmer this is not related to Next.js at all, React expects a list of class names separated by spaces exactly like HTML

**Reply by pedrocardoz0** [reply]
That's cool

**Comment by JohnFoxtrot** [comment]
I've had success with this:

```
<div className={`${styles.first} ${styles.second} ${styles.etc}`}>
```
